vendor/assets/stylesheets/dvl/components/page_header.scss

```diff
@@ -33,35 +36,92 @@
   }
 
   > .button,
-  > form .button {
+  > form .button,
```

Comment:
I know this is here for a reason. I'm fine with removing it if we check every page in Screendoor...